Snakefile

```diff
@@ -1,10 +1,13 @@
+from snakemake.utils import validate
+
 PYTHON = "PYTHONPATH=./ python"
 PYTHON_SCRIPT = "PYTHONPATH=./ python {input} {output}"
 PYTHON_SCRIPT_WITH_CONFIG = PYTHON_SCRIPT + " {CONFIG_FILE}"
 
-CONFIG_FILE = "config/default.yaml"
+CONFIG_FILE = "config/default{}.yaml"
+configfile: CONFIG_FILE.format("")
+validate(config, CONFIG_FILE.format(".schema"))
```

Comment:
Hm, I don't really understand this. The schema should be always the same, no matter which config file used, no? So shouldn't the schema be called `schema.yaml`?

Reply:
yeah, fair, I'll update